app/controllers/api/v1/hosts_controller.rb

```diff
@@ -1,6 +1,9 @@
 module Api
   module V1
     class HostsController < V1::BaseController
+      include Api::CompatibilityChecker
+      before_filter :check_create_host_nested, :only => [:create]
```

Comment:
> Applicable for :update too?

Pushed, without unit test tho, don't think it's necessary.

-- 
Later,
 Lukas #lzap Zapletal